_layouts/default.html

```diff
@@ -4,15 +4,18 @@
 
   <body>
 
-    <div>
-      <a href="{{ site.baseurl }}/">
-        <header>
-          <img src="{{ site.baseurl }}/img/logo.svg" alt="{{ site.title }}" />
-        </header>
-      </a>
-
-      {{ content }}
-
+    <div id="wrapper">
+      <div id="content">
+        <div>
```

Comment:
Yes, I am, it's to ensure the footer is at the bottom of the page, even if the page is not taking all the height. If you have a better solution, I will gladly try it